packages/design-system/src/components/web-components/ds-button/ds-button.test.tsx

```diff
@@ -123,52 +120,78 @@ describe('Button', () => {
       (window as any as UtagContainer).utag = {
         link: tealiumMock,
       };
+      jest.useFakeTimers();
```

Comment:
Is the goal here to import the shared functions defined in `analyticsTesting.ts`?

Reply:
Yeah, I was experimenting with those shared functions, and I think I'll end up changing the way they're structured so they're easier to reuse. Whatever the result of that, I'll test them consistently between components